Dockerfile

```diff
@@ -138,7 +138,7 @@ RUN curl -kSs https://pyenv.run | bash \
 ARG CONAN_VERSION=1.57.0
 ARG PYTHON_INSPECTOR_VERSION=0.9.8
 ARG PYTHON_PIPENV_VERSION=2022.9.24
-ARG PYTHON_POETRY_VERSION=1.2.2
+ARG PYTHON_POETRY_VERSION=1.6.1
```

Comment:
Please upgrade the Poetry version in Dockerfile-legacy as well. Dockerfile-legacy is used by CI, maybe that is related to the failing tests.

Reply:
@mnonnenmacher: I hope you are fine with changing python-inspector to 0.9.8 as well, as this will resolve: https://github.com/oss-review-toolkit/ort/issues/7333#issuecomment-1667728868

Reply:
Yes please, versions should in general stay in sync. Installing Python 3.11 would be more difficult with the legacy file, so I think it's fine to stay with the default 3.10 there. python-inspector should still support 3.11 as long as no setup.py file which depends on 3.11 is involved.